src/app/query/bitcoin/ordinals/utils.ts

```diff
@@ -32,31 +32,19 @@ export function getTaprootAddress({ index, keychain, network }: GetTaprootAddres
   return payment.address;
 }
 
-// In lieu of reliable API, we scrape HTML from the Ordinals.com explorer and
-// parses the HTML
-// Example:
-// https://ordinals.com/output/758bd2703dd9f0a2df31c2898aecf6caba05a906498c9bc076947f9fc4d8f081:0
-async function getOrdinalsComTxOutputHtmlPage(id: string, index: number) {
-  const resp = await fetch(`https://ordinals-explorer.generative.xyz/output/${id}:${index}`);
-  const html = await resp.text();
-  return new DOMParser().parseFromString(html, 'text/html');
+interface UtxoItem {
```

Comment:
If you look here there is already a type for this: https://github.com/hirosystems/wallet/blob/089001662a120fd9b03c43a86000fccf9e688bef/src/app/query/bitcoin/ordinals/use-ordinals-aware-utxo.query.ts#L18

And, I think you can just relocate this and use the query hook? I don't think this needs to be a separate utility now.

Reply:
Or, looks like should be it's own query hook now?

Reply:
hm, if I use hook, it would be rather big refactor, may be I can just use that ready `getOrdinalsAwareUtxo` function?

Reply:
I pushed the fix